src/helix-ui/elements/HXFileTileElement.less

```diff
@@ -180,3 +180,9 @@
     display: block;
   }
 }
+
+:host([readonly]) {
+  #hxDismiss {
+    display: none;
+  }
```

Comment:
Make sure to apply padding all the way around the `#hxContentWrapper`, when readonly.

Reply:
Padding added to `#hxContentWrapper` to make it `0.5rem` all around, when readonly:

```css
:host([readonly]) {
  #hxDismiss {
    display: none;
  }

  #hxContentWrapper {
    padding: 0.5rem;
  }
}
```